nengo/config.py

```diff
@@ -84,6 +84,10 @@ def __repr__(self):
         return "<%s[%s]{%s}>" % (type(self).__name__,
                                  self._configures.__name__, ", ".join(params))
 
+    def __setstate__(self, state):
+        for k, v in state.items():
+            setattr(self, k, v)
```

Comment:
What about raising an AttributeError rather than calling the super `__getattr__`? This also seems to work for me. Since `__getattr__` is only ever called as a fallback if the attribute isn't found normally, this doesn't seem problematic to me.